apps/src/templates/projects/ProjectCard.jsx

```diff
@@ -35,9 +205,217 @@ export default class ProjectCard extends React.Component {
+                  {checkboxes.map(checkbox => (
+                    <label
+                      key={checkbox.name}
+                      style={{
+                        display: 'flex',
+                        alignItems: 'center',
+                        marginTop: 5,
+                        marginBottom: 5,
+                      }}
+                    >
+                      <input
+                        type="checkbox"
+                        checked={checkbox.checked}
+                        onChange={() =>
+                          this.handleCheckboxChange(checkbox.name)
+                        }
+                        style={{
+                          marginRight: 10,
+                          marginTop: 0,
+                          accentColor: '#0093A4',
+                        }}
+                      />
+                      <span
+                        style={{
+                          flex: '1',
+                          ...styles.popUpBody,
+                          verticalAlign: 'middle',
+                        }}
+                      >
+                        {checkbox.name.replace(/-/g, ' ')}
+                      </span>
+                    </label>
+                  ))}
+                </div>
+                {isAnyCheckboxSelected ? (
+                  <div
+                    style={{padding: 5}}
+                    className="g-recaptcha"
+                    data-sitekey={captchaSiteKey}
+                    data-callback="onCaptchaSubmit"
+                    data-expired-callback="onCaptchaExpired"
+                  />
+                ) : null}
+                <hr style={styles.popUpLines} />
+                <div style={{display: 'flex', justifyContent: 'flex-end'}}>
+                  <Button
+                    onClick={this.cancel}
+                    text={'Cancel'}
+                    color={Button.ButtonColor.white}
+                    style={{
+                      borderColor: color.neutral_dark,
+                      color: color.neutral_dark,
+                    }}
+                  />
+                  <Button
+                    onClick={this.handleSubmit}
+                    disabled={!submitButtonEnabled}
+                    text={'Submit'}
+                    style={{outline: 'none'}}
+                    color={Button.ButtonColor.brandSecondaryDefault}
+                  />
+                </div>
+              </div>
+            )}
+          </AccessibleDialog>
+        ) : null}
+
         <div style={styles.card}>
+          {!showSubmittedHeader ? (
+            <div
+              style={{
+                ...thumbnailStyle,
+                height: 40,
+                display: 'flex',
+                alignItems: 'center',
+                justifyContent: 'flex-end',
+              }}
+            >
+              <button
+                type="button"
+                onClick={this.showReportAbusePopUp}
+                style={{
```

Comment:
You can simplify this code by using scss module instead of inline styles. This way you won't need that much state variables and callbacks

Reply:
That makes it a lot simpler, thank you!